### ISSUE TITLE
Enable semantic support for other languages and extensions

### ISSUE THREAD

**Issue by yourmentorsf** [goliafrs/monokai#7]
Not hard to do. Just add to lines to your config. This enables theme support for dib notebooks as well. 

>   "type": "dark",
>   "semanticHighlighting": true,

```json
{
  "name": "Monokai GRS",
  "type": "dark",  // desirable
  "semanticHighlighting": true, // required
  "tokenColors": [

  ]
}
```

**Comment by goliafrs** [comment]
[v1.0.12](https://github.com/goliafrs/monokai/releases/tag/v1.0.12)

**Comment by goliafrs** [comment]
After thorough testing, we decided not to enable the `semanticHighlighting` parameter. We found that it caused most of the code to be highlighted in unusual colors, which was confusing.

In the future, we plan to create a separate version of this theme with `semanticHighlighting` enabled. This version will be customized separately to ensure the color highlighting is intuitive and helpful.

Updates about this new version will be posted in this repository's description. If you're interested, you'll be able to switch to this version once it's available.
